eplaunch/utilities/locateworkflows.py

```diff
@@ -31,13 +33,21 @@ def find(self):
         return self.list_of_found_directories
 
     def get_energyplus_versions(self):
+        ep_names = {
+            Platform.WINDOWS: 'energyplus.exe',
+            Platform.LINUX: 'energyplus',
+            Platform.MAC: 'energyplus',
+            Platform.UNKNOWN: ''
+        }
         for workspace_directory in self.list_of_found_directories:
             energyplus_directory, folder_name = os.path.split(workspace_directory)
-            energyplus_application = os.path.join(energyplus_directory, "energyplus.exe")
+            energyplus_application = os.path.join(energyplus_directory, ep_names[Platform.get_current_platform()])
             if os.path.exists(energyplus_application):
                 self.list_of_energyplus_applications.append(energyplus_application)
                 found, version_string, build_string = self.get_specific_version_from_exe(energyplus_application)
-                if not found:
+                if found:
+                    self.list_of_energyplus_versions.append({'version': version_string, 'sha': build_string})
```

Comment:
I'm not **actually** sure what you wanted to store for each version, but I figured a little dictionary would suffice for now.  It allows testing it anyway.